docs/zh/latest/admin-api.md

```diff
@@ -449,140 +575,177 @@ service 对象 json 配置内容：
 }
 ```
 
-具体示例：
+### 使用示例：
```

Comment:
```suggestion
### 使用示例
```

Reply:
Note: when using Chinese as a Heading, I suggest adding an English hash.

I forget how to set custom anchor 😅

```suggestion
### 使用示例 {xxxx}
```

Reply:
ok, i'll study it.

Reply:
https://docusaurus.io/docs/markdown-features/toc#heading-ids